src/quo/components/wallet/token_input/view.cljs

```diff
@@ -37,22 +44,28 @@
           {:style  style/token
            :source (resources/get-token token)}]
          [rn/text-input
-          {:ref                    #(reset! input-ref %)
-           :placeholder            "0"
-           :placeholder-text-color (colors/theme-colors colors/neutral-40 colors/neutral-50 theme)
-           :keyboard-type          :numeric
-           :max-length             12
-           :default-value          @value
-           :on-change-text         #(reset! value %)
-           :style                  (style/text-input theme)
-           :selection-color        customization-color}]
+          (cond-> {:ref                    #(do
+                                              (reset! input-ref %)
+                                              (when %
+                                                (.focus ^js %)))
+                   :placeholder            "0"
+                   :placeholder-text-color (colors/theme-colors colors/neutral-40
+                                                                colors/neutral-50
+                                                                theme)
+                   :keyboard-type          :numeric
+                   :max-length             12
+                   :on-change-text         #(reset! value %)
+                   :style                  (style/text-input theme)
+                   :selection-color        customization-color
+                   :showSoftInputOnFocus   show-keyboard?}
```

Comment:
can be kebab-case -> show-soft-input-on-focus ?

Reply:
Fixed